release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -370,6 +381,15 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-configuring-power-states-using-ztp"]
+==== Configuring power states using zero touch provisioning (ZTP)
+
+{product-title} {product-version} introduces the ability to configure using zero touch provisioning (ZTP) the various power state configurations for critical and non-critical workloads.
```

Comment:
Please reword this to indicate that OCP 4.12 had introduced the ability to set power states for critical and non-critical workloads. This release now enables the user to configure power states using ZTP.